app/code/community/Vindi/Subscription/Model/PaymentMethod.php

```diff
@@ -78,7 +78,7 @@ public function assignData($data)
 	/**
 	 * @return string
 	 */
-	protected function getPaymentMethodCode()
+	public function getPaymentMethodCode()
```

Comment:
Pq alterar a visibilidade desses métodos?

Reply:
Estamos alterando a visibilidade dos métodos por conta do fato de que o Magento desencoraja a utilizar métodos `protected` ou `private`, levando em conta esse princípio [aqui](https://en.wikipedia.org/wiki/Composition_over_inheritance). Para aplicar isso no módulo por completo, teríamos que mexer em praticamente o módulo todo, o que não é a ideia desse PR (talvez no futuro)

Além disso, não é possível testar métodos que não sejam públicos, e a ideia é garantir que os mesmos façam o que devem fazer para os próximos PRs que for abrir

Reply:
show, perfeito :D